lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -933,8 +932,8 @@ class ADContext {
     return generatedFunctions;
   }
 
-  SmallVector<SILValue, 32> &getGeneratedDerivativeFunctionReferences() {
-    return generatedDerivativeFunctionReferences;
+  SmallVector<SILValue, 32> &getGeneratedFunctionReferences() {
```

Comment:
Note: function references to generated subset parameter thunks (for linear map and derivative functions) are not tracked. This is not a problem because all references to such thunks are in other generated functions, so cleanup already deletes all such references.

I decided not to track references to subset parameter thunks for now, since there's no problem.